tests/fixtures/dataset.py

```diff
@@ -36,7 +36,7 @@
     "content": str,
     "internal_links": list,
     "external_links": list,
-    "source": str,
+    "source": tuple,
```

Comment:
Why a tuple

Reply:
That's the output given by `info["source"] = (self.__class__.__name__,)`

Reply:
why is info['source'] a tuple, and not a string?

Reply:
This is just the way you had it, I can un-tuple it

Reply:
bahah i don't remember why i did it anymore